edxval/migrations/0002_data__default_profiles.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+DEFAULT_PROFILES = [
+    dict(profile_name="desktop_mp4",  width=1280, height= 720, extension="mp4"),
+    dict(profile_name="desktop_webm", width=1280, height= 720, extension="webm"),
+    dict(profile_name="mobile_high",  width= 960, height= 540, extension="mp4"),
+    dict(profile_name="mobile_low",   width= 640, height= 360, extension="mp4"),
+    dict(profile_name="youtube",      width=1920, height=1080, extension="mp4"),
+]
+
+
+def create_default_profiles(apps, schema_editor):
+    Profile = apps.get_model("edxval", "Profile")
+    for kwargs in DEFAULT_PROFILES:
+        Profile.objects.get_or_create(**kwargs)
+
+
+def delete_default_profiles(apps, schema_editor):
+    Profile = apps.get_model("edxval", "Profile")
+    for kwargs in DEFAULT_PROFILES:
+        Profile.objects.get_or_create(**kwargs).delete()
```

Comment:
If you `create()` and `delete()` in the same filter, does it do both in the DB? Or is it smart enough to do nothing?

I think it's fine either way - data migration is a one-time, out-of-band thing.